Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -109,7 +116,13 @@ for i in ${!Subjects[@]}; do
     --subject="$Subject" \
     --path="$StudyFolder" \
     --sessions="$TPlist" \
-    --template-id="$LongitudinalTemplate")
+    $fslsub_queue_param \
```

Comment:
newopts will happily take `--fslsub_queue="" ` or `--fslsub_queue= `, you don't need to add complications to leave the parameter out of the command line.
```suggestion
    --fslsub_queue="$QUEUE" \
```

Reply:
Ok, fixed.